tests/test_black.py

```diff
@@ -1352,6 +1352,39 @@ def test_parse_pyproject_toml(self) -> None:
         self.assertEqual(config["exclude"], r"\.pyi?$")
         self.assertEqual(config["include"], r"\.py?$")
 
+    def test_parse_pyproject_toml_project_metadata(self) -> None:
```

Comment:
This would be a good place for pytest parameterise instead of a loop!

Reply:
That would be great, but unfortunately, `parametrize` doesn't work within this `BlackTestCase` class. It's a different testing framework.

Not sure why the tests are set up like this - there are different test files that do use normal pytest functions and parametrize - but for now I have to resort to this loop 😞

Reply:
Oh that's right. Well it's fine this way as well.